CHANGELOG.md

```diff
@@ -1,601 +1,181 @@
-# Treasure Agent 4 changelog
+# Fluent Package 5 changelog
 
-## Release v4.5.0 - 2023/05/08
+About the past changelog entries, see [old CHANGELOG](CHANGELOG-v4.md) instead.
+
+## Release v5.0.0 - 2023/07/DD
 
 ### News
 
-* Update bundled Ruby to 2.7.8 which is the last version of Ruby 2.7 series.
-  * Exceptionally for RHEL 9 & Ubuntu 22.04 jammy, update Ruby to 3.1.4.
-  * The next major version of td-agent which bundles Ruby 3.2 is still work in
-    progress. We'll announce it as soon as we are ready for it.
-    The package name will be changed to `fluent-package`.
-* Update fluentd to 1.16.1
-* Update fluent-plugin-kafka to 0.19.0
-* Update fluent-plugin-elasticsearch to 5.3.0
-* Update fluent-plugin-opensearch to 1.1.0
-* Update nokogiri to 1.14.3 (Windows)
-* Drop Debian buster support
+* `td-agent` is renamed to `fluent-package`. (#448,#449,#463,#518)
+  * This represents current community-oriented development styles well.
+* Debian 12 (bookworm) has been supported. (#462,#509)
+* Removed Ubuntu 16.04 (xenial), Ubuntu 18.04 (bionic) support. (#457,#509)
+* Amazon Linux 2023 has been supported. (#459)
+* Introduced new package signing key. The new key will be used in the future
+  release. we still use using old signing key for a while. (#507)
+
+Not only changing package name, but also there are some notable changes.
+Basically, for `td-agent` v4 users, it aims to keep compatibility as far as possible
+by executing the migration process with copying old files or providing
+symbolic links for it. For newly installed `fluent-package`, no such
+compatibility layer will not provided. (It means that fresh install should not provides
+such a redundant backward compatible symbolic links and so on)
```

Comment:
> For newly installed `fluent-package`, no such ...

I think this description might be unnecessary.

Reply:
It may be too negative wording.
Thanks, fixed it.